### PR TITLE
Link

### PR DESCRIPTION
**Pull request by mani87**

### CODE REVIEW

**Comment by mani87** [comment]
@codehimanshu Can you see this please?
Thankyou:p

File: ui/events.html [hunk 1]
```diff
@@ -103,17 +103,17 @@
...
 							<ul>
 								<li>
-									<a class="" href="/">Home</a>
+									<a class="" href="index.html">Home</a>
 								</li>
 
 
 								<li>
-									<a class="" href="/events.html">Events</a>
+									<a class="" href="events.html">Events</a>
```
Comment (codehimanshu) [inline ui/events.html]: I think href="#" would be even better. The page will not reload. In this case it might reopen the same link.

**Comment by codehimanshu** [comment]
Why does this PR contain **#35  404 page displayed according to UI c7199c0** 
Can you please squash and remove these commits from the PR?

**Comment by mani87** [comment]
@codehimanshu rebased them all.

**Comment by codehimanshu** [comment]
I think the #35 is still linking here and squash both of you r commits. also use -f to push

**Comment by mani87** [comment]
@codehimanshu Okay Done :)

**Comment by codehimanshu** [comment]
@mani87 Thanks for you contribution.

**Pull request merged**